build.sbt

```diff
@@ -38,7 +37,8 @@ libraryDependencies ++= Seq(
   libraryDependencies ++= Seq(
     "org.apache.spark" % "spark-core_2.10" % sparkVersion excludeAll(excludeHadoop),
     "org.apache.spark" % "spark-mllib_2.10" % sparkVersion excludeAll(excludeHadoop),
-    "org.apache.spark" % "spark-sql_2.10" % sparkVersion excludeAll(excludeHadoop)
+    "org.apache.spark" % "spark-sql_2.10" % sparkVersion excludeAll(excludeHadoop),
+    "edu.berkeley.cs.amplab" % "mlmatrix_2.10" % "0.1" excludeAll(excludeHadoop)
   )
```

Comment:
So when things conflict maven and SBT have different resolution rules AFAIK. One of them picks the higher version while the other picks the first thing it sees. Which is which I forget